src/GlobalStyle.js

```diff
@@ -8,21 +8,24 @@ const GlobalStyle = createGlobalStyle`
     font-family: Montserrat, sans-serif;
     font-weight: 700;
     font-size: 16px;
-    overflow-y: scroll;
   }
 
   html,
   body {
-    min-height: 100vh;
+    height: 100%;
+    width: 100%;
   }
 
   .page {
     display: flex;
     flex-direction: column;
-    justify-content: center;
+    justify-content: flex-start;
     align-items: center;
     flex-grow: 1;
-    padding-top: 22px;
+    position: relative;
+    min-height: 100vh;
+    padding: 22px 0 100px;
```

Comment:
This padding is probably very specific to the Home page, right? You can add another styled component to the Home page that [extends the Page styles](https://styled-components.com/docs/basics#extending-styles)